### PR TITLE
Fixes #15649: At installation Rudder-webapp only creates rudder-slapd and ncf-api-venv users but do not force group creation

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/15649

### CODE REVIEW

**Comment by Fdall** [comment]
PR updated with a new commit

File: rudder-webapp/SOURCES/rudder-webapp-postinst [hunk 1]
```diff
@@ -12,16 +12,24 @@ LDAP_CONF="/opt/rudder/etc/openldap/slapd.conf"
...
-  useradd --system --shell /bin/false --home-dir /var/rudder/ldap --comment "Rudder LDAP server,,," rudder-slapd >> ${LOG_FILE}
+  useradd --system --gid rudder-slapd --shell /bin/false --home-dir /var/rudder/ldap --comment "Rudder LDAP server,,," rudder-slapd >> ${LOG_FILE}
 fi
 chown root:rudder-slapd "${LDAP_CONF}"
 chmod 640 "${LDAP_CONF}"
 chown -R rudder-slapd:rudder-slapd /var/rudder/ldap/
 
 if ! getent passwd ncf-api-venv >/dev/null; then
-  useradd --system --shell /bin/false --home-dir /var/lib/ncf-api-venv --groups rudder --comment "ncf API,,," ncf-api-venv >> ${LOG_FILE}
+  useradd --system --gid rudder-slapd --shell /bin/false --home-dir /var/lib/ncf-api-venv --groups rudder --comment "ncf API,,," ncf-api-venv >> ${LOG_FILE}
```
Comment (peckpeck) [inline rudder-webapp/SOURCES/rudder-webapp-postinst]: probably not

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
OK, squash merging this PR

**Pull request merged**